pandas/tests/series/test_constructors.py

```diff
@@ -148,6 +149,30 @@ def test_constructor_categorical(self):
         self.assertTrue(is_categorical_dtype(s))
         self.assertTrue(is_categorical_dtype(s.dtype))
 
+    def test_constructor_categorical_dtype(self):
+        result = pd.Series(['a', 'b'],
+                           dtype=pd.CategoricalDtype(['a', 'b', 'c'],
+                                                     ordered=True))
+        assert is_categorical_dtype(result) is True
+        tm.assert_index_equal(result.cat.categories, pd.Index(['a', 'b', 'c']))
+        assert result.cat.ordered
+
+        result = pd.Series(['a', 'b'], dtype=pd.CategoricalDtype(['b', 'a']))
+        assert is_categorical_dtype(result)
+        tm.assert_index_equal(result.cat.categories, pd.Index(['b', 'a']))
+        assert result.cat.ordered is False
```

Comment:
if you are asserting that the result is a boolean (``False``) , then the first is more appropriate. If its just not ``True`` then the 2nd is fine (e.g. imagine 0 satisfies 2nd but not 1st). In this case we *do* want to assert a boolean result.